Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/admin.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/admin.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
"GET /static/admin/css/login.css HTTP/1.1" 304 0
"GET /static/admin/css/responsive.css HTTP/1.1" 304 0
"GET /static/admin/css/base.css HTTP/1.1" 304 0
"GET /static/admin/css/fonts.css HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
"POST /admin/login/?next=/admin/ HTTP/1.1" 200 1980
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
"POST /admin/login/?next=/admin/ HTTP/1.1" 200 1980
"POST /admin/login/?next=/admin/ HTTP/1.1" 200 1980
"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 4606
"GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
"GET /admin/account/profile/ HTTP/1.1" 200 3180
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
"GET /static/admin/js/urlify.js HTTP/1.1" 304 0
"GET /static/admin/js/core.js HTTP/1.1" 304 0
"GET /static/admin/js/actions.js HTTP/1.1" 304 0
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
"GET /static/admin/css/changelists.css HTTP/1.1" 304 0
"GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
"GET /admin/ HTTP/1.1" 200 4606
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/forms.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/forms.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/forms.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /admin/account/profile/ HTTP/1.1" 200 3180
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/account/profile/add/ HTTP/1.1" 200 5798
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/change_form.js HTTP/1.1" 304 0
"GET /static/admin/css/forms.css HTTP/1.1" 304 0
"GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 304 0
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
"GET /static/admin/js/calendar.js HTTP/1.1" 304 0
"GET /static/admin/css/widgets.css HTTP/1.1" 304 0
"GET /static/admin/img/icon-calendar.svg HTTP/1.1" 304 0
"GET /static/admin/img/calendar-icons.svg HTTP/1.1" 200 1094
"POST /admin/account/profile/add/ HTTP/1.1" 302 0
"GET /admin/account/profile/ HTTP/1.1" 200 5059
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
"GET /admin/account/ HTTP/1.1" 200 2455
"GET /admin/account/profile/ HTTP/1.1" 200 4857
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/account/profile/add/ HTTP/1.1" 200 5798
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"POST /admin/account/profile/add/ HTTP/1.1" 302 0
"GET /admin/account/profile/ HTTP/1.1" 200 5465
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/account/ HTTP/1.1" 200 2455
"GET /admin/account/profile/ HTTP/1.1" 200 5263
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/account/profile/2/change/ HTTP/1.1" 200 6340
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/ HTTP/1.1" 200 5218
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /account/edit/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py", line 31, in edit
    profile_form = ProfileEditForm(instance=request.user.profile)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/utils/functional.py", line 257, in inner
    return func(self._wrapped, *args)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/db/models/fields/related_descriptors.py", line 415, in __get__
    self.related.get_accessor_name()
django.contrib.auth.models.User.profile.RelatedObjectDoesNotExist: User has no profile.
"GET /account/edit/ HTTP/1.1" 500 78352
"GET /account/login/ HTTP/1.1" 200 1537
"GET /account/login/ HTTP/1.1" 200 1537
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 815
"GET /account/edit/ HTTP/1.1" 200 1990
"GET /account/logout/ HTTP/1.1" 200 631
"GET /account/login/ HTTP/1.1" 200 1306
"GET /account/login/ HTTP/1.1" 200 1306
"POST /account/login/ HTTP/1.1" 200 1398
"GET /account/login/ HTTP/1.1" 200 1306
"GET /account/register/ HTTP/1.1" 200 1532
"POST /account/register/ HTTP/1.1" 200 636
"GET /account/login/ HTTP/1.1" 200 1306
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 812
"GET /account/edit/ HTTP/1.1" 200 1738
"POST /account/edit/ HTTP/1.1" 200 1774
"GET /admin/account/profile/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/account/profile/ HTTP/1.1" 200 2003
"POST /admin/login/?next=/admin/account/profile/ HTTP/1.1" 302 0
"GET /admin/account/profile/ HTTP/1.1" 200 5578
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/logout/ HTTP/1.1" 200 631
"GET /account/logout/ HTTP/1.1" 200 631
"GET /account/login/ HTTP/1.1" 200 1306
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 938
"GET /account/edit/ HTTP/1.1" 200 1993
"GET /account/password_change/ HTTP/1.1" 200 1775
"GET /account/logout/ HTTP/1.1" 200 631
"GET /account/login/ HTTP/1.1" 200 1306
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 942
"GET /static/css/base.css HTTP/1.1" 304 0
"GET /account/edit/ HTTP/1.1" 200 1997
"POST /account/edit/ HTTP/1.1" 200 2172
"GET /account/edit/ HTTP/1.1" 200 1999
"POST /account/edit/ HTTP/1.1" 200 2170
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/admin.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/admin.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /account/logout/ HTTP/1.1" 200 635
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
"GET /static/admin/css/login.css HTTP/1.1" 304 0
"GET /static/admin/css/responsive.css HTTP/1.1" 304 0
"GET /static/admin/css/base.css HTTP/1.1" 304 0
"GET /static/admin/css/fonts.css HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 5889
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
"GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /account/login/ HTTP/1.1" 200 1541
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 942
Not Found: /images/create/
"GET /images/create/?title=%20Django%20and%20Duke&url=http://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 404 2536
Not Found: /images/create/
"GET /images/create/?title=%20Django%20and%20Duke&url=https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg HTTP/1.1" 404 2521
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /images/create/Django and Duke/https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg/
"GET /images/create/Django%20and%20Duke/https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg/ HTTP/1.1" 404 2753
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /images/create/Django and Duke/
"GET /images/create/Django%20and%20Duke/ HTTP/1.1" 404 2463
"GET /account/login/ HTTP/1.1" 200 1544
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /images/create/
"GET /images/create/ HTTP/1.1" 404 2393
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/ HTTP/1.1" 200 1510
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /images/create/Django and Duke/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
TypeError: image_create() got an unexpected keyword argument 'title'
"GET /images/create/Django%20and%20Duke/ HTTP/1.1" 500 66989
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/Django%20and%20Duke/ HTTP/1.1" 200 1510
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/Django%20and%20Duke/https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg HTTP/1.1" 301 0
"GET /images/create/Django%20and%20Duke/https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg/ HTTP/1.1" 200 1510
"GET /images/create/Django%20and%20Duke/https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg/ HTTP/1.1" 200 1510
"GET /account/ HTTP/1.1" 200 942
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg HTTP/1.1" 200 1576
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg HTTP/1.1" 200 1576
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg HTTP/1.1" 200 1576
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg HTTP/1.1" 200 1600
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://commons.wikimedia.org/wiki/File:Django_Reinhardt_and_Duke_Ellington_(Gottlieb).jpg HTTP/1.1" 200 1576
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1600
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1600
Internal Server Error: /images/create/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 25, in image_create
    return redirect(new_item.get_absolute_url())
AttributeError: 'Image' object has no attribute 'get_absolute_url'
"POST /images/create/ HTTP/1.1" 500 72322
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 2090
"GET /static/admin/css/login.css HTTP/1.1" 304 0
"GET /static/admin/css/base.css HTTP/1.1" 304 0
"GET /static/admin/css/responsive.css HTTP/1.1" 304 0
"GET /static/admin/css/fonts.css HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 5889
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
"GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
"GET /static/admin/js/core.js HTTP/1.1" 304 0
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/css/changelists.css HTTP/1.1" 304 0
"GET /static/admin/js/urlify.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
"GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
"GET /static/admin/js/actions.js HTTP/1.1" 304 0
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
"GET /admin/images/image/1/change/ HTTP/1.1" 200 8756
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/css/forms.css HTTP/1.1" 304 0
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
"GET /static/admin/js/change_form.js HTTP/1.1" 304 0
"GET /static/admin/css/widgets.css HTTP/1.1" 304 0
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /admin/images/image/1/delete/ HTTP/1.1" 200 3208
"GET /static/admin/js/cancel.js HTTP/1.1" 200 409
"POST /admin/images/image/1/delete/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 3995
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/images/ HTTP/1.1" 200 2434
"GET /admin/images/image/ HTTP/1.1" 200 3840
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/images/ HTTP/1.1" 200 2434
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1597
Internal Server Error: /images/create/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 25, in image_create
    return redirect(new_item.get_absolute_url())
AttributeError: 'Image' object has no attribute 'get_absolute_url'
"POST /images/create/ HTTP/1.1" 500 72367
"GET /admin/images/ HTTP/1.1" 200 2550
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"POST /admin/images/image/ HTTP/1.1" 200 3332
"POST /admin/images/image/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 3961
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1597
Internal Server Error: /images/create/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 25, in image_create
    return redirect(new_item.get_absolute_url())
AttributeError: 'Image' object has no attribute 'get_absolute_url'
"POST /images/create/ HTTP/1.1" 500 72351
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1766
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /admin/images/ HTTP/1.1" 200 2434
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"POST /admin/images/image/ HTTP/1.1" 200 3332
"POST /admin/images/image/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 3961
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1597
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1597
"GET /admin/images/ HTTP/1.1" 200 2434
"GET /admin/images/image/ HTTP/1.1" 200 3840
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1597
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1597
Watching for file changes with StatReloader
"GET /account/logout/ HTTP/1.1" 200 635
"GET /account/login/ HTTP/1.1" 200 1310
"GET /admin/images/image/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/images/image/ HTTP/1.1" 200 1845
"POST /admin/login/?next=/admin/images/image/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 3840
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
Forbidden (CSRF token missing or incorrect.): /account/login/
"POST /account/login/ HTTP/1.1" 403 2513
"GET /account/login/ HTTP/1.1" 200 1541
"GET /account/login/ HTTP/1.1" 200 1541
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 942
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1600
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /images/create/ HTTP/1.1" 200 1510
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1600
Internal Server Error: /images/create/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 26, in image_create
    return redirect(new_item.get_absolute_url())
AttributeError: 'Image' object has no attribute 'get_absolute_url'
"POST /images/create/ HTTP/1.1" 500 73579
"GET /admin/images/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/images/ HTTP/1.1" 200 2104
"GET /admin/login HTTP/1.1" 301 0
"GET /admin/login/ HTTP/1.1" 200 1806
"GET /static/admin/css/login.css HTTP/1.1" 200 1233
"GET /static/admin/css/responsive.css HTTP/1.1" 200 17944
"GET /static/admin/css/base.css HTTP/1.1" 200 16378
"GET /static/admin/css/fonts.css HTTP/1.1" 200 423
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2316
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 85692
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
"POST /admin/login/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 6639
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 86184
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6918
"GET /static/admin/js/core.js HTTP/1.1" 200 7099
"GET /static/admin/js/urlify.js HTTP/1.1" 200 8941
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 271817
"GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1530
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
"GET /static/admin/js/actions.js HTTP/1.1" 200 6766
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
"POST /admin/images/image/ HTTP/1.1" 200 3332
"GET /static/admin/js/cancel.js HTTP/1.1" 200 409
"POST /admin/images/image/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 3961
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
"GET /admin/images/image/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/images/image/ HTTP/1.1" 200 2000
"GET /admin/login/?next=/admin/images/image/ HTTP/1.1" 200 2000
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1600
Internal Server Error: /images/create/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 23, in image_create
    messages.success(request, 'Image added successfully')
AttributeError: 'Image' object has no attribute 'get_absolute_url'
"POST /images/create/ HTTP/1.1" 500 72460
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"POST /admin/images/image/ HTTP/1.1" 200 3332
"POST /admin/images/image/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 3961
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1769
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1600
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/create/?title=%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg HTTP/1.1" 200 1600
Internal Server Error: /images/create/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 25, in image_create
    return redirect(new_item.get_absolute_url())
AttributeError: 'Image' object has no attribute 'get_absolute_url'
"POST /images/create/ HTTP/1.1" 500 72357
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/forms.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/images/ HTTP/1.1" 200 2434
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
"GET /account/ HTTP/1.1" 200 1600
"GET /account/edit/ HTTP/1.1" 200 1997
"GET /account/ HTTP/1.1" 200 1431
"GET /account/password_change/ HTTP/1.1" 200 1779
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /static/css/base.css HTTP/1.1" 200 4515
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2316
"GET /account/ HTTP/1.1" 200 1431
"GET /static/css/base.css HTTP/1.1" 304 0
"GET /account/logout/ HTTP/1.1" 200 635
"GET /account/login/ HTTP/1.1" 200 1310
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 1431
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin HTTP/1.1" 301 0
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
"GET /static/admin/css/login.css HTTP/1.1" 200 1233
"GET /static/admin/css/responsive.css HTTP/1.1" 200 17944
"GET /static/admin/css/base.css HTTP/1.1" 200 16378
"GET /static/admin/css/fonts.css HTTP/1.1" 200 423
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 85692
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2316
"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 7139
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 86184
"GET /admin/images/image/ HTTP/1.1" 200 5606
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
"GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6918
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/core.js HTTP/1.1" 200 7099
"GET /static/admin/js/actions.js HTTP/1.1" 200 6766
"GET /static/admin/js/urlify.js HTTP/1.1" 200 8941
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 271817
"GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1530
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
"GET /admin/images/image/6/change/ HTTP/1.1" 200 8842
"GET /static/admin/js/change_form.js HTTP/1.1" 200 712
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/css/forms.css HTTP/1.1" 200 8518
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
"GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
"GET /static/admin/img/tooltag-arrowright.svg HTTP/1.1" 200 280
"POST /admin/images/image/6/change/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 5943
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
"GET /admin/images/image/6/change/ HTTP/1.1" 200 9038
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"POST /admin/images/image/6/change/ HTTP/1.1" 302 0
"GET /admin/images/image/ HTTP/1.1" 200 5885
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /account/ HTTP/1.1" 200 1431
Not Found: /bookmars
"GET /bookmars HTTP/1.1" 404 2307
Not Found: /bookmarks
"GET /bookmarks HTTP/1.1" 404 2310
Not Found: /bookmarks/images/create/
"GET /bookmarks/images/create/ HTTP/1.1" 404 2355
"GET /images/create/ HTTP/1.1" 200 1510
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/detail/6/django-and-duke HTTP/1.1" 301 0
Internal Server Error: /images/detail/6/django-and-duke/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endblock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 36, in image_detail
    return render(request, 'images/image/detail.html', {'section': 'images', 'image': image})
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 156, in __init__
    self.nodelist = self.compile_nodelist()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 194, in compile_nodelist
    return parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 266, in do_extends
    nodelist = parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 209, in do_block
    nodelist = parser.parse(('endblock',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/sorl/thumbnail/templatetags/thumbnail.py", line 195, in thumbnail
    return ThumbnailNode(parser, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/sorl/thumbnail/templatetags/thumbnail.py", line 121, in __init__
    self.nodelist_file = parser.parse(('empty', 'endthumbnail',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 13: 'endblock', expected 'empty' or 'endthumbnail'. Did you forget to register or load this tag?
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 500 169999
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 200 963
"GET /media/cache/72/14/72145c523c07ddff3042e37681abea8a.jpg HTTP/1.1" 200 47931
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /images/detail/6/django-and-duke/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endblock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 38, in image_detail
    return render(request, 'images/image/detail.html', {'section': 'images', 'image': image})
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 156, in __init__
    self.nodelist = self.compile_nodelist()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 194, in compile_nodelist
    return parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 266, in do_extends
    nodelist = parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 209, in do_block
    nodelist = parser.parse(('endblock',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/defaulttags.py", line 1472, in do_with
    nodelist = parser.parse(('endwith',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 34: 'endblock', expected 'endwith'. Did you forget to register or load this tag?
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 500 170101
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 200 3198
"GET /media/cache/72/14/72145c523c07ddff3042e37681abea8a.jpg HTTP/1.1" 304 0
"GET /static/css/base.css HTTP/1.1" 304 0
"POST /images/like/ HTTP/1.1" 200 16
"POST /images/like/ HTTP/1.1" 200 16
"POST /images/like/ HTTP/1.1" 200 16
"POST /images/like/ HTTP/1.1" 200 16
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 200 3201
"POST /images/like/ HTTP/1.1" 200 16
"POST /images/like/ HTTP/1.1" 200 16
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/common/decorators.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/like HTTP/1.1" 301 0
Bad Request: /images/like/
"GET /images/like/ HTTP/1.1" 400 0
Bad Request: /images/like/
"GET /images/like/ HTTP/1.1" 400 0
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2316
Watching for file changes with StatReloader
Not Found: /images/
"GET /images/ HTTP/1.1" 404 2754
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2316
"GET /account/ HTTP/1.1" 200 3368
"GET /static/css/base.css HTTP/1.1" 304 0
"GET /account/ HTTP/1.1" 200 3368
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /account/users/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py", line 77, in user_list
    'users': users})
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: account/user/list.html
"GET /account/users/ HTTP/1.1" 500 86051
"GET /account/users/ HTTP/1.1" 200 3950
"GET /media/cache/c4/52/c4526ab211d7071410806ec4d977130e.jpg HTTP/1.1" 200 15077
"GET /static/css/base.css HTTP/1.1" 304 0
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 200 13723
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1974
"GET /static/admin/css/login.css HTTP/1.1" 304 0
"GET /static/admin/css/responsive.css HTTP/1.1" 200 17944
"GET /static/admin/css/base.css HTTP/1.1" 200 16378
"GET /static/admin/css/fonts.css HTTP/1.1" 304 0
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 85692
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 86184
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
"GET /static/admin/css/base.css HTTP/1.1" 200 16378
"GET /static/admin/css/login.css HTTP/1.1" 200 1233
"GET /static/admin/css/responsive.css HTTP/1.1" 200 17944
"GET /static/admin/css/fonts.css HTTP/1.1" 200 423
Not Found: /favicon.ico
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
"GET /favicon.ico HTTP/1.1" 404 2316
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 85692
"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 7164
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
"GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 86184
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /admin/images/image/ HTTP/1.1" 200 5696
"GET /static/admin/js/core.js HTTP/1.1" 200 7099
"GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6918
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1530
"GET /static/admin/js/actions.js HTTP/1.1" 200 6766
"GET /static/admin/js/urlify.js HTTP/1.1" 200 8941
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 271817
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
"GET /admin/ HTTP/1.1" 200 7164
"GET /admin/auth/user/ HTTP/1.1" 200 7902
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/img/search.svg HTTP/1.1" 200 458
"GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
"GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
"GET /admin/auth/user/2/change/ HTTP/1.1" 200 15125
"GET /static/admin/js/calendar.js HTTP/1.1" 200 7777
"GET /static/admin/css/forms.css HTTP/1.1" 200 8518
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 20218
"GET /static/admin/js/SelectBox.js HTTP/1.1" 200 5753
"GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 12322
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
"GET /static/admin/js/change_form.js HTTP/1.1" 200 712
"GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
"GET /static/admin/img/tooltag-arrowright.svg HTTP/1.1" 200 280
"GET /static/admin/img/icon-unknown.svg HTTP/1.1" 200 655
"GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 200 655
"GET /static/admin/img/selector-icons.svg HTTP/1.1" 200 3291
"GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
"GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
"GET /admin/auth/user/ HTTP/1.1" 200 7902
"GET /admin/auth/user/2/change/ HTTP/1.1" 200 15125
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/auth/user/ HTTP/1.1" 200 7902
"GET /admin/auth/ HTTP/1.1" 200 2968
"GET /admin/ HTTP/1.1" 200 7164
"GET /admin/images/image/ HTTP/1.1" 200 5696
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/ HTTP/1.1" 200 7164
"GET /admin/auth/user/ HTTP/1.1" 200 7902
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/auth/user/2/change/ HTTP/1.1" 200 15125
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /account/users/ HTTP/1.1" 200 3950
"GET /media/cache/c4/52/c4526ab211d7071410806ec4d977130e.jpg HTTP/1.1" 304 0
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 304 0
Internal Server Error: /account/users/flaugher/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/defaulttags.py", line 513, in render
    return self.nodelist.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 176, in render
    template = context.template.engine.get_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 129, in find_template
    raise TemplateDoesNotExist(name, tried=tried)
django.template.exceptions.TemplateDoesNotExist: images/image/list_ajax.html

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py", line 68, in user_detail
    'user': user})
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 63, in render
    reraise(exc, self.backend)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 84, in reraise
    raise new from exc
django.template.exceptions.TemplateDoesNotExist: images/image/list_ajax.html
"GET /account/users/flaugher/ HTTP/1.1" 500 174388
"GET /account/users/flaugher/ HTTP/1.1" 200 3590
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 200 46315
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 304 0
"GET /account/users/ HTTP/1.1" 200 3950
"GET /media/cache/c4/52/c4526ab211d7071410806ec4d977130e.jpg HTTP/1.1" 304 0
"GET /account/users/michaels/ HTTP/1.1" 200 3183
"GET /account/users/ HTTP/1.1" 200 3950
"GET /account/users/flaugher/ HTTP/1.1" 200 3590
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 304 0
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /account/users/flaugher/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py", line 71, in user_detail
    'user': user})
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 156, in __init__
    self.nodelist = self.compile_nodelist()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 194, in compile_nodelist
    return parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 266, in do_extends
    nodelist = parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 209, in do_block
    nodelist = parser.parse(('endblock',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 209, in do_block
    nodelist = parser.parse(('endblock',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 205, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'domready' appears more than once
"GET /account/users/flaugher/ HTTP/1.1" 500 178222
"GET /account/users/flaugher/ HTTP/1.1" 200 4031
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 304 0
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 304 0
"GET /account/users/flaugher/ HTTP/1.1" 200 3099
"GET /account/users/flaugher/ HTTP/1.1" 200 3098
"GET /static/css/base.css HTTP/1.1" 200 4515
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 200 46315
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 200 13723
"GET /account/users/flaugher/ HTTP/1.1" 200 3085
"GET /static/css/base.css HTTP/1.1" 200 4515
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 200 13723
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 200 46315
Internal Server Error: /account/users/follow/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/common/decorators.py", line 14, in wrap
    return f(request, *args, **kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/views/decorators/http.py", line 40, in inner
    return func(request, *args, **kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py", line 78, in user_follow
    user_id = require_POST.get('id')
AttributeError: 'function' object has no attribute 'get'
"POST /account/users/follow/ HTTP/1.1" 500 15529
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /account/users/flaugher/ HTTP/1.1" 200 3085
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 200 13723
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 200 46315
"GET /static/css/base.css HTTP/1.1" 200 4515
"POST /account/users/follow/ HTTP/1.1" 200 16
"POST /account/users/follow/ HTTP/1.1" 200 16
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/admin.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/admin.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1974
"GET /admin/actions/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/actions/ HTTP/1.1" 200 1990
"POST /admin/login/?next=/admin/actions/ HTTP/1.1" 302 0
"GET /admin/actions/ HTTP/1.1" 200 2450
"GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
"GET /admin/ HTTP/1.1" 200 7847
"GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
"GET /admin/actions/action/ HTTP/1.1" 200 4355
"GET /static/admin/js/core.js HTTP/1.1" 200 7099
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6918
"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
"GET /static/admin/css/changelists.css HTTP/1.1" 304 0
"GET /static/admin/js/actions.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
"GET /static/admin/img/search.svg HTTP/1.1" 304 0
"GET /static/admin/js/urlify.js HTTP/1.1" 200 8941
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 271817
"GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
"GET /admin/actions/action/add/ HTTP/1.1" 200 6625
"GET /static/admin/js/change_form.js HTTP/1.1" 304 0
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/css/forms.css HTTP/1.1" 200 8518
"GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/actions/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /admin/ HTTP/1.1" 200 7847
"GET /admin/actions/action/ HTTP/1.1" 200 4355
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /admin/ HTTP/1.1" 200 7847
"GET /account/ HTTP/1.1" 302 0
"GET /account/login/?next=/account/ HTTP/1.1" 200 3256
"GET /static/css/base.css HTTP/1.1" 200 4515
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2316
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 3382
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /account/ HTTP/1.1" 200 3451
"GET /account/users/ HTTP/1.1" 200 3950
"GET /media/cache/11/8e/118ea974c14f4a46f33e52e6395f42c8.jpg HTTP/1.1" 200 13723
"GET /media/cache/c4/52/c4526ab211d7071410806ec4d977130e.jpg HTTP/1.1" 200 15077
"GET /account/users/michaels/ HTTP/1.1" 200 2678
"POST /account/users/follow/ HTTP/1.1" 200 16
"GET /account/ HTTP/1.1" 200 3451
"GET /account/users/ HTTP/1.1" 200 3950
"GET /account/users/shari/ HTTP/1.1" 200 2588
"GET /account/ HTTP/1.1" 200 3451
Internal Server Error: /account/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/account/views.py", line 38, in dashboard
    { 'section': 'dashboard', 'actions': actions })
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/defaulttags.py", line 309, in render
    return nodelist.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/defaulttags.py", line 443, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/urls/base.py", line 90, in reverse
    return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/urls/resolvers.py", line 673, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'list' not found. 'list' is not a valid view function or pattern name.
"GET /account/ HTTP/1.1" 500 153723
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /account/ HTTP/1.1" 200 3458
Internal Server Error: /images/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 76, in image_list
    paginator = Paginator(images, 8)
NameError: name 'Paginator' is not defined
"GET /images/ HTTP/1.1" 500 69757
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/ HTTP/1.1" 200 3186
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 200 46315
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 200 3222
"GET /media/cache/72/14/72145c523c07ddff3042e37681abea8a.jpg HTTP/1.1" 200 47931
"POST /images/like/ HTTP/1.1" 200 16
"POST /images/like/ HTTP/1.1" 200 16
"GET /account/ HTTP/1.1" 200 3458
"GET /admin/logout/ HTTP/1.1" 200 2572
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/apps.py changed, reloading.
Watching for file changes with StatReloader
"GET /account/ HTTP/1.1" 302 0
"GET /account/login/?next=/account/ HTTP/1.1" 200 3256
"GET /static/css/base.css HTTP/1.1" 304 0
"POST /account/login/ HTTP/1.1" 302 0
"GET /account/ HTTP/1.1" 200 3458
"GET /images/ HTTP/1.1" 200 3186
"GET /media/cache/33/ba/33ba998b7880eeaf30c6d6b480940271.jpg HTTP/1.1" 200 46315
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 200 3222
"GET /media/cache/72/14/72145c523c07ddff3042e37681abea8a.jpg HTTP/1.1" 200 47931
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
"GET /static/admin/css/login.css HTTP/1.1" 200 1233
"GET /static/admin/css/responsive.css HTTP/1.1" 200 17944
"GET /static/admin/css/base.css HTTP/1.1" 200 16378
"GET /static/admin/css/fonts.css HTTP/1.1" 200 423
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2316
"GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
"GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 85692
"POST /admin/login/?next=/admin/ HTTP/1.1" 200 1980
"GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 86184
"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 7847
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
"GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /admin/images/image/ HTTP/1.1" 200 5696
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/actions.js HTTP/1.1" 200 6766
"GET /static/admin/js/core.js HTTP/1.1" 200 7099
"GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1530
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6918
"GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 271817
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
"GET /static/admin/js/urlify.js HTTP/1.1" 200 8941
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
"GET /admin/images/image/6/change/ HTTP/1.1" 200 9504
"GET /static/admin/js/change_form.js HTTP/1.1" 200 712
"GET /admin/jsi18n/ HTTP/1.1" 200 3223
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
"GET /static/admin/css/forms.css HTTP/1.1" 200 8518
"GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
"GET /static/admin/img/tooltag-arrowright.svg HTTP/1.1" 200 280
"POST /images/like/ HTTP/1.1" 200 16
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/bookmarks/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/ HTTP/1.1" 200 3186
Internal Server Error: /images/detail/6/django-and-duke/
Traceback (most recent call last):
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py", line 57, in image_detail
    'total_views': total_views})
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 156, in __init__
    self.nodelist = self.compile_nodelist()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 194, in compile_nodelist
    return parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 266, in do_extends
    nodelist = parser.parse()
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/loader_tags.py", line 209, in do_block
    nodelist = parser.parse(('endblock',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/defaulttags.py", line 1472, in do_with
    nodelist = parser.parse(('endwith',))
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 449, in parse
    raise self.error(token, e)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 447, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 563, in compile_filter
    return FilterExpression(token, self)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 657, in __init__
    filter_func = parser.find_filter(filter_name)
  File "/Users/flaugher/.virtualenvs/django2_by_example/lib/python3.7/site-packages/django/template/base.py", line 569, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'plurarlize'
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 500 181182
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 200 3238
"GET /images/detail/6/django-and-duke/ HTTP/1.1" 200 3238
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/flaugher/projects/django/django2_by_example/bookmarks/images/urls.py changed, reloading.
Watching for file changes with StatReloader
"GET /images/ranking HTTP/1.1" 301 0
"GET /images/ranking/ HTTP/1.1" 200 2771
